components/teams-service/server/v2/server_test.go

```diff
@@ -983,7 +983,58 @@ func runAllServerTests(
 			}
 		})
 
-		t.Run("fails to add user with NotFound when a project filter that excludes the team", func(t *testing.T) {
+		t.Run("successfully adds user to team with existing users", func(t *testing.T) {
```

Comment:
realized we were missing this test scenario. all the other tests add users to teams with no users